projects/packages/post-list/src/class-post-list.php

```diff
@@ -80,43 +67,43 @@ public function enqueue_scripts( $hook ) {
 				plugin_dir_url( __DIR__ ) . 'build/style.rtl.css'
 			);
 
-			wp_set_script_translations( 'jetpack_posts_list_ui_script', 'jetpack' );
-
-			add_action( 'admin_footer', array( $this, 'print_post_data' ) );
+			add_filter( 'manage_posts_columns', array( $this, 'add_thumbnail_column' ) );
```

Comment:
I wonder if we should move these into the `init` action, along with the check that we're on the right admin page.

Reply:
Do you mean, like adding something like `add_action( 'init', array( $this, 'thumbnail_filter_action' ) );` to `register()`? I don't think 'init' returns a $hook variable to check if we're on 'edit.php'

Reply:
Yeah I wasn't sure about adding the other hooks in the enqueue scripts one, but if it's needed.

I think you can check `current_screen` during `admin_init`. Might be worth checking, but don't worry about it too much.

Reply:
I didn't have much luck with admin_init. But I moved them out of admin_enqueue_scripts and into current_screen. Seems a bit more logical.